src/main/scala/io/iohk/ethereum/validators/BlockHeaderValidator.scala

```diff
@@ -18,6 +21,14 @@ object BlockHeaderValidatorImpl {
   val MaxPowCaches: Int = 2 // maximum number of epochs for which PoW cache is stored in memory
 
   class PowCacheData(val cache: Array[Int], val dagSize: Long)
+
+  // FIXME: this is used to speed up ETS Blockchain tests. All blocks in those tests have low numbers (1, 2, 3 ...)
```

Comment:
I think this temp fix is ok. Did you create a YT task for it?

Reply:
EC-331